# Handling numbers

### Add two integers

In [4]:
2+2

4

### Substract two integers

In [5]:
6-4

2

### Divide two integers

In [6]:
9/3

3.0

Note that integer division results in a decimal

In [8]:
9/2

4.5

### Multiply two integers

In [7]:
5*4

20

### Add two decimals

In [10]:
18.0+7.0

25.0

In [11]:
18.0+7

25.0

In [12]:
18.2+7.4

25.6

### Substract two decimal

In [14]:
18.2-7.4

10.799999999999999

Note that the correct answer is 10.8, but the answer from python is a close approximation

### Multiply two decimals

In [15]:
18.2*7.4

134.68

### Divide two decimals

In [16]:
18.2/7.4

2.459459459459459

### Modulus (Remainder of divisio operation) of two integers

In [18]:
7%2

1

### Modulus (Remainder of divisio operation) of two decimals

In [22]:
18.2/7.4

2.459459459459459

In [23]:
18.2%7.4

3.3999999999999986

In [24]:
7.4*2

14.8

In [25]:
18.2-14.8

3.3999999999999986

### Exponent operation

In [26]:
2**3

8

In [27]:
-5**2

-25

In [28]:
(-5)**2

25